angular-workspace/projects/ni/nimble-angular/table-column/anchor/nimble-table-column-anchor.directive.ts

```diff
@@ -35,14 +35,6 @@ export class NimbleTableColumnAnchorDirective extends NimbleTableColumnBaseDirec
         this.renderer.setProperty(this.elementRef.nativeElement, 'hrefFieldName', value);
```

Comment:
Looking at the chromatic tests I think this case looks extra weird because all values on the row are undefined. But my question is when you group on an undefined / absent value, how should the group header render?

![image](https://github.com/ni/nimble/assets/1588923/551ece45-0046-4bbe-9675-263a3a4487e8)

This probably applied before too if placeholder was not set, guess that just wasn't covered in tests.

Reply:
We also may have multiple potential "blank" groupings, i.e. number columns would represent different NaN values as blank

Reply:
Yeah, I would not consider this in scope for this PR, but it's a fair question. My implementation for the mapping columns printed the actual record value in the group header. This was to address the scenario with multiple blank group headers. We could do the same for number/date columns if we want. When the record value is actually missing/null, we could make the header show something like "*Blank* (1)". That's probably better than what we currently have.

Reply:
Agree that it's not in scope for this PR and was a pre-existing concern just unmasked with these changes. Also think the cases for a string column are something like: null, undefined, empty string, and string with various white space. Think all of those would look weird as group headers.

The advice today is to pre-format your data to avoid those cases. Can you create a discussion issue for us to gather client feedback around the behavior for the future? Then we can link to that and resolve the comment for this PR.

Reply:
Created #1367